packages/frontend/src/components/svg/ArrowIcon.js

```diff
@@ -2,12 +2,13 @@ import React from 'react';
 
 const ArrowIcon = ({ color }) => {
     const stroke = color || '#0072CE';
+
     return (
-        <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <svg width='24' height='24' viewBox='0 0 24 24' fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
nit: can we please skip this change to keep it consistent

Reply:
kk